src/hooks/useSimpleLayout.js

```diff
@@ -0,0 +1,65 @@
+
+export const useSimpleLayout = props => {
+  const {
+    columns,
+    rows,
+    hooks: {
+      columns: columnsHooks,
+      getRowProps,
+      getHeaderRowProps,
+      getHeaderProps,
+      getCellProps
+    }
+  } = props
+
+  rows.forEach((row, i) => {
+    if (row.cells === undefined) {
+      row.cells = columns.map(column => {
+        const cell = {
```

Comment:
Shouldn't cell generation happen regardless of what renderer or hook we use? I thought it was part of the `prepare` script.

Reply:
If cell generation is happening in `useFlexLayout`, then I don't blame you for putting it here. It might be better to put that in the `prepareRow` function and let hooks plug in to that. Thoughts?